docker-images/kaniko-executor/Makefile

```diff
@@ -0,0 +1,14 @@
+PROJECT_NAME := kaniko-executor
+KANIKO_EXECUTOR = gcr.io/kaniko-project/executor:v1.3.0
```

Comment:
why it's not `latest`?

Reply:
I think we should have a fixed tag and when we decide to upgrade it we should do that intentionally with proper testing etc. Not just because Kaniko changes the latest to something else.